public/app/features/alerting/unified/components/export/providers.ts

```diff
@@ -32,12 +43,51 @@ export const HclExportProvider: ExportProvider<'hcl'> = {
   type: 'terraform',
 };
 
+export const OperatorExportProvider: ExportProvider<'operator'> = {
+  name: 'Grafana Operator',
+  exportFormat: 'operator',
+  overrideLanguage: 'yaml',
+  type: 'operator',
+  supports: ['AlertRuleGroup', 'AlertRule', 'Receiver', 'RuleFolder'],
+  formatter: (raw: string) => {
+    const out: any[] = [];
+    const parsed = loadAll(raw) as Array<{ kind: string }>;
+    for (const resource of parsed) {
+      console.log(resource);
+      if (resource.kind === 'GrafanaAlertRuleGroup') {
+        const r = resource as any;
```

Comment:
creating this variable with an any type should be unnecessary after the piping changes above